app/navigation/Navigator.scala

```diff
@@ -251,6 +251,13 @@ class Navigator @Inject()() {
     case _ => getCheckYourAnswers
   }
 
+  private val repayRouteMap: Page => UserAnswers => Call = {
+    case CustomsDutyPaidPage => _ => routes.RepaymentAmountSummaryController.onPageLoad(CheckMode)
+    case VATPaidPage => _ => routes.RepaymentAmountSummaryController.onPageLoad(CheckMode)
+    case OtherDutiesPaidPage => _ => routes.RepaymentAmountSummaryController.onPageLoad(CheckMode)
+    case _ => getCheckYourAnswers
+  }
```

Comment:
Can we have some unit tests for these scenarios (navigatorSpec)?

Reply:
Thanks Lorenzo, added the test cases.